getndazn/pensieve
# fix: updated npm dependencies (security audit)

# Description

I updated some of the NPM dependencies as they raised a security issue. 
One of them established a breaking change:
"axios": from "^0.19.2" to "^0.21.1"
This change should not pose any risk on production code as Axios is a dev dependency used in the `examples` folder (to test the library locally).

## How to verify this change

Resolution can be verified by:
- Checkout in this feature branch: `git checkout npm-dependencies-update`
- Install the NPM dependencies and verify in the output that no security issue is flagged: `npm i`


### Related issues

-

# PR status

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

# Checklist

- [x] I have *commented* my code where necessary, particularly in hard-to-understand areas
- [x] The *code coverage* hasn't dropped
